apps/evm/lib/evm/logger.ex

```diff
@@ -0,0 +1,60 @@
+defmodule EVM.Logger do
+  require Logger
+  alias EVM.{Operation, MachineState}
+
+  @doc """
+  This function logs state in the same format as Parity's `evm-debug` function. This makes comparing implementations and debugging easier.
+
+  `cargo test --features "json-tests evm/evm-debug-tests" --release -- BlockchainTests_GeneralStateTest_stSystemOperationsTest --nocapture`
+  """
+
+  @spec log_state_in_parity_format(EVM.Operation.Metadata.t(), MachineState.t()) :: nil
+  def log_state_in_parity_format(operation, machine_state) do
```

Comment:
As @germsvel mentioned above this method too big and can be divided

Reply:
ah, yep! Done: [Refactor `EVM.Logger`](https://github.com/poanetwork/mana/pull/164/commits/29a6ef8f809f4703c46f6716b8d1e636d569e9f6)